docs/source/examples/pec_tutorial.myst

```diff
@@ -0,0 +1,311 @@
+
+$$
+U = \sum_i a_i G_i,
+$$
+is replaced by $G_i$ with probability $|a_i| /\gamma$, where $\gamma=\sum_i |a_i|$.
+
+This procedure is repeated multiple times, generating $M$ (`num_samples`) circuits. The associated noisy expectation values are re-combined by Mitiq according to the following formula
+
+$$
+\frac{\gamma}{M} \sum_{i=1}^M {\rm sign}(a_i) E_i ,
+$$
+
+which is an unbiased estimator of the true (noiseless) expectation value.
+
+```{code-cell} ipython3
+cnot_counts, oneq_counts = [], []
+true_values, noisy_values = [], []
+pec_values = []
+
+noise_scaled_expectation_values = []
+
+for depth in depths:
+    print("Status: On depth", depth, end="\n\n")
+    
+    cnot_depth_counts, oneq_depth_counts = [], []
+    true_depth_values, noisy_depth_values = [], []
+    pec_depth_values = []
+
+    noise_scaled_expectation_depth_values = []
+
+    for trial in range(trials):
+        # Local seed is calculated in this way to ensure that we don't get repeat values in loop.        
+        local_seed = 10**6 * depth + 10**3 * seed + trial
+        
+        circuit, correct_bitstring = get_circuit(depth, local_seed)
+
+        true_value, = execute(circuit, ideal_backend, shots, correct_bitstring, is_noisy=False)
+        noisy_value, = execute(circuit, noisy_backend, shots, correct_bitstring, is_noisy=True)
+        
+        pec_executor = functools.partial(
+            execute, 
+            backend=noisy_backend, 
+            shots=shots // num_samples,
+            correct_bitstring=correct_bitstring,
+        )
+        
+        pec_value = pec.execute_with_pec(
+            circuit, 
+            pec_executor, 
+            representations=get_representations(computer), 
+            num_samples=num_samples,
+            random_state=local_seed,
+        )
+
+        pec_depth_values.append(pec_value)
+
+        cnot_depth_counts.append(get_num_cnot_count(circuit))
+        oneq_depth_counts.append(get_oneq_count(circuit))
+        
+        true_depth_values.append(true_value)
+        noisy_depth_values.append(noisy_value)
+    
+    cnot_counts.append(cnot_depth_counts)
+    oneq_counts.append(oneq_depth_counts)
+    
+    true_values.append(true_depth_values)
+    noisy_values.append(noisy_depth_values)
+    
+    noise_scaled_expectation_values.append(noise_scaled_expectation_depth_values)    
+    
+    pec_values.append(pec_depth_values)
+```
+
+
+Now we can visualize the results.
+
+```{code-cell} ipython3
+avg_true_values = np.average(true_values, axis=1)
+avg_noisy_values = np.average(noisy_values, axis=1)
+
+std_true_values = np.std(true_values, axis=1, ddof=1)
+std_noisy_values = np.std(noisy_values, axis=1, ddof=1)
+
+avg_pec_values = np.average(pec_values, axis=1)
+std_pec_values = np.std(pec_values, axis=1, ddof=1)
+
+plt.figure(figsize=(9, 5))
+
+plt.plot(depths, avg_true_values, label="True", lw=2)
+plt.errorbar(depths, avg_noisy_values, yerr=std_noisy_values, label="Raw")
```

Comment:
```suggestion
eb = plt.errorbar(depths, avg_noisy_values, yerr=std_noisy_values, label="Raw")
eb[-1][0].set_linestyle('-.')
```
This or some slight variation might work, based on discussion at https://stackoverflow.com/a/23002214/20670981 .